org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/html/HtmlHelper.java

```diff
@@ -90,13 +91,16 @@ public String toPlainTextNoTrim(String html) {
       // <body> not found, use entire input
       s = html;
     }
+    //remove comments
+    Matcher matcher = HTML_COMMENT.matcher(s);
+    s = matcher.replaceAll("");
+    //remove attribute values since they could contain special characters like >
+    s = removeAttributeValues(s);
     //newlines
     s = StringUtility.replace(s, "\r", "");
     s = StringUtility.replace(s, "\n", " ");
-    Matcher matcher = HTML_PARAGRAPH_END_TAGS.matcher(s);
+    matcher = HTML_PARAGRAPH_END_TAGS.matcher(s);
     s = matcher.replaceAll("\n");
-    //tabs
-    s = StringUtility.replace(s, StringUtility.HTML_ENCODED_TAB, "\t");
```

Comment:
Is it correct to remove this? If we remove this, can we also delete the HTML_ENCODED_TAB constant?